templates/etc_archivematica_archivematicaCommon_dbsettings.j2

```diff
@@ -2,4 +2,4 @@
 user=archivematica
 password=demo
 host=localhost
-db=MCP
+database=MCP
```

Comment:
Can we leave both of these in?  That way all branches deployed with an updated role this don't need the fix.